core/src/main/java/org/opensearch/sql/executor/CanPaginateVisitor.java

```diff
@@ -44,13 +44,13 @@ public Boolean visitRelation(Relation node, Object context) {
     return Boolean.TRUE;
   }
 
+  /*
```

Comment:
Is canPaginate not used anywhere?

Reply:
It is used in commented code below only.